go.mod

```diff
@@ -6,5 +6,5 @@ require (
 	github.com/gin-gonic/gin v1.7.7
 	github.com/shirou/gopsutil v3.20.11+incompatible
 	github.com/stretchr/testify v1.7.0
-	mosn.io/pkg v0.0.0-20211217101631-d914102d1baf
+	mosn.io/pkg latest
```

Comment:
I don't think it's a good idea to use `latest` here, use a specified version will be a better choice.
otherwise, break changes from mosn.io/pkg may break holmes.